src/background/tokenBackground.ts

```diff
@@ -118,6 +180,28 @@ export default class TokenBackground {
     }
   }
 
+  private addToken = async (contractAddress: string, name: string, symbol: string, decimals: number) => {
+    const newToken = new QRCToken(name, symbol, decimals, contractAddress);
+    this.tokens!.push(newToken);
+    this.setTokenListInChromeStorage();
+    await this.getQRCTokenBalance(newToken);
+  }
+
+  private removeTokenAtIndex = (index: number) => {
+    this.tokens!.splice(index, 1);
+    this.setTokenListInChromeStorage();
+  }
+
+  private setTokenListInChromeStorage = () => {
+    chrome.storage.local.set({
+      [this.chromeStorageAccountTokenListKey()]: this.tokens,
+    }, () => console.log('accountTokenList', this.tokens));
+  }
+
+  private chromeStorageAccountTokenListKey = () => {
+    return STORAGE.ACCOUNT_TOKEN_LIST.concat(this.bg.account.loggedInAccount!.name).concat(this.bg.network.networkName);
```

Comment:
add separator to make the key more readable (should we ever need to debug the storage):
`${STORAGE.ACCOUNT_TOKEN_LIST}-${this.bg.account.loggedInAccount!.name}-${this.bg.network.networkName}`